samples/high_throughput_with_dask/dump_test_output.py

```diff
@@ -0,0 +1,9 @@
+import pickle
+import base64
+import sys
+
+base64_data = sys.stdin.read()
+pickle_data = base64.b64decode(base64_data)
+data = pickle.loads(pickle_data)
+
+print(data)
```

Comment:
Is the print statement intentional here? If not, as it is a test, should we try to assert something?

Reply:
I removed this file entirely. It was there for testing but didn't use the dask_espresso module for decoding.